src/Inkluzitron/Data/BotDatabaseContext.cs

```diff
@@ -50,6 +53,32 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     from => from.ToString("yyyy-MM-dd"),
                     to => DateTime.ParseExact(to, "yyyy-MM-dd", null)
                 ));
+
+            var ulongStringConverter = new ValueConverter<ulong, string>(
+                from => from.ToString(),
+                to => ulong.Parse(to)
+            );
+
+            modelBuilder
+                .Entity<ScheduledTask>()
+                .Property(i => i.When)
+                .HasConversion(new ValueConverter<DateTimeOffset, string>(
+                    from => from.ToUniversalTime().ToString("O"),
+                    to => DateTimeOffset.ParseExact(to, "O", CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal)
+                ));
+
+            modelBuilder.Entity<VoteReplyRecord>().Property(i => i.GuildId).HasConversion(ulongStringConverter);
```

Comment:
By mě zajímalo, co z toho vyplodí migrace. To udělá sloupec se stringem?

Reply:
Však tam máš soubory migrace, se můžeš podívat 😄
Ale jinak jo, bude to string